.gitignore

```diff
@@ -1,13 +1,9 @@
 # See https://help.github.com/ignore-files/ for more about ignoring files.
 
-# dependencies
 /node_modules
-
-# testing
 /coverage
-
-# production
 /build
+/docs
```

Comment:
If GitHub pages requires `/docs`, then we can't ignore it, right?

Reply:
Yes!